### ISSUE TITLE
autowaf.build_pc generates lilv-0.pc which is missing serd-0, sord-0, and sratom-0 dependency

### ISSUE THREAD

**Issue by SpotlightKid** [drobilla/lilv#30]
The current build produces a `lilv-0.pc` pkg-config file , which looks like this:

```
prefix=/usr
exec_prefix=${prefix}
libdir=${exec_prefix}/lib
includedir=${prefix}/include

Name: Lilv
Version: 0.24.5
Description: Simple C library for hosting LV2 plugins
Requires: lv2   
Libs: -L${libdir} -llilv-0  -ldl
Cflags: -I${includedir}/lilv-0
```

This is missing `serd-0`, `serd-0` and `sratom-0` in the `Requires:` line.

I think something must have changed in the `autowaf.build_pc` function when `waflib` was included as a git submodule instead of as a binary blob in the `waf` script, so the substitution of `@PKG_serd_0@ @PKG_sord_0@ @PKG_sratom_0@` in the `lilv.pc.in`  no longer works correctly.

**Comment by SpotlightKid** [comment]
Or the way `autowaf.build_pc()` has to be called might have changed.

**Comment by SpotlightKid** [comment]
I'm using this workaround in the `lilv-git` AUR package: https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=lilv-git#n42

**Comment by drobilla** [comment]
Oops.

Fixed in 79a4549, thanks.
